actionmailbox/app/controllers/rails/conductor/action_mailbox/incinerates_controller.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+module Rails
+  # Incinerating will destroy an email that is due and has already been processed.
+  class Conductor::ActionMailbox::IncineratesController < Rails::Conductor::BaseController
+    def create
```

Comment:
Shouldn't the action be `destroy` handled by a `DELETE` request? 🤔

Reply:
If the resource was the `InboundMessage` I would agree that it's a `destroy` of the `InboundMessage`, but in this controller the resource is the `Incineration`, so we want to `create` a new incineration for the inbound message. Does that makes sense?